MaxText/max_utils.py

```diff
@@ -219,6 +219,9 @@ def maybe_initialize_jax_distributed_system(raw_keys):
 
   For CPUs, we call jax.distributed.initialize() explicitly, with the specified arguments.
   """
+  if raw_keys["skip_jax_distributed_system"]:
```

Comment:
Nit: Thanks for the log! So we know the reason to skip the initialization. Could you help add accordingly logs to following 2 cases? `inference_benchmark_test` & `compile_topology`?